compiler/test/formatter_inputs/includes.gr

```diff
@@ -0,0 +1,32 @@
+module Includes
+
+include "runtime/unsafe/memory"
+include "runtime/unsafe/tags"
+include "list"
+include "option" as Opt
```

Comment:
```suggestion
include "option"
  as Opt
```
Is this valid at the parser? Ideally these formatter_inputs would have weird formatting to verify that the output is nice instead of "didn't change anything"

Reply:
Nah, no breaks within this statement are supported.

Reply:
Really? That seems like an arbitrary decision and it means that line wrapping is totally broken/unavailable for include statements. e.g.
```
include "my/really/really/really/really/really/extremely/deep/path" as SomethingThatShouldWrap
```

Reply:
It's more because line breaks aren't currently allowed when aliasing something than an intentional decision. https://github.com/grain-lang/grain/issues/1636